class-tgm-plugin-activation.php

```diff
@@ -451,6 +451,8 @@ public function init() {
 			if ( true === $this->has_forced_deactivation ) {
 				add_action( 'switch_theme', array( $this, 'force_deactivation' ) );
 			}
+
+			add_action( 'admin_head',  array( $this, 'tgm_css' ), 21 );
```

Comment:
It was set to 21 as because if someone wants to add other CSS formatting with low or higher priority. But, I removed the priority.